prow/config.yaml

```diff
@@ -933,6 +933,78 @@ periodics:
         path: /mnt/disks/ssd0
       name: cache-ssd
 
+- name: ci-kubernetes-e2e-gce-ubuntu-node
+  interval: 30m
+  spec:
+    containers:
+    - args:
+      - --timeout=50
+      - --bare
+      - --gce-ssh=/etc/ssh-key-secret/ssh-private
+      - --gce-pub=/etc/ssh-key-secret/ssh-public
```

Comment:
the args here is for bootstrap.py, not the scenarios. The args for the scenarios are from config.json.